node/src/proxy_server/mod.rs

```diff
@@ -3849,33 +3856,36 @@ mod tests {
                 assert!(!proxy_server.stream_key_routes.is_empty());
                 assert!(!proxy_server.tunneled_hosts.is_empty());
                 TestLogHandler::new().exists_log_containing(&format!(
-                    "DEBUG: {test_name}: Last data received for stream key zpi6SfvohvBgqq9y8zLgzIhxPS4, \
-                    which was tunneling through host Some(\"hostname\"). It will be purged after 500ms."
+                    "DEBUG: {test_name}: Client closed stream referenced by stream key {:?}, \
+                    which was tunneling to the host Some(\"hostname\"). \
```

Comment:
This won't block approval, but in case you felt energetic, this could be made friendlier.